en/homework/advanced/otp-concurrency/otp-concurrency.md

```diff
@@ -0,0 +1,23 @@
+# OTP concurrency / GenServer homework
+
```

Comment:
What do you think about using the tests like a guide, sort of like TDD? In other words, we instruct students to run the tests at the beginning, rather than only if they get stuck. This has the added benefit of giving students the chance to get familiar with running Elixir tests, reading test output and using that output to guide their decisions to get the tests passing. 

Something like:

To get started, run the tests by executing `elixir otp-concurrency.exs`. The instructions below will tell you what you need to do to complete the homework assignment and get the tests passing.

Reply:
@SophieDeBenedetto I think this is _great_. I can't speak for @wkulikowski but I for one am not :100: how to work this in in a non-invasive way and without reworking all the lessons to fit it. Do you have any suggestions or example material I could look at for ideas?

Reply:
@doomspork not totally sure what you're asking here? How to work what in in a non-invasive way?

Reply:
Sorry, the TDD approach you described.

Reply:
Ah got it 👍 

I'm not sure why adopting this approach would require re-working existing lessons? I'm thinking of this more as a framing devise for homework assignments specifically.

Reply:
Also 💯 for @SophieDeBenedetto I didn't put much thought into wording and assumed that tests = TDD. Will change that